config/locales/es.yml

```diff
@@ -1407,207 +1407,203 @@ es:
-                  Ten cuidado con las estafas. Si parece demasiado bueno para ser cierto,
-                  probablemente lo es. Y no mandes dinero a alguien que no conoces.
+                  No haga clic en enlaces cuyos orígenes no esté 100% seguro. Tenga cuidado con las estafas.
+                  Si parece demasiado bueno para ser verdad, probablemente lo sea. Y no envíe dinero a
+                  alguien que no conoce. Mire las tarifas que se cobran si está pagando su servicio de
+                  preparación de impuestos o un préstamo de anticipación de reembolso al hacer que se deduzcan
+                  de su reembolso.
           how_to_get_paid:
-            header: Cómo recibir el pago
+            header: ¿Cómo cobrar?
             items:
-              - title: "¿Cómo obtengo mi Pago de Impacto Económico?"
+              - title: "¿Cómo puedo recibir mi cheque del Pago de Impacto Económico?"
                 content_html: >-
-                  Si el IRS aún no te ha enviado el pago de la primera o de la
-                  segunda ronda de estímulo, o si recibiste un monto incorrecto
-                  para cualquiera de los dos pagos, deberás presentar una
-                  declaración de impuestos para el año 2020 para recibirlo.
-              - title: "¿Cómo sé si el IRS envió mi cheque de estímulo?"
+                  Si el IRS aún no ha enviado su primera o segunda ronda de pagos de estímulo,
+                  o si recibió una cantidad incorrecta para cualquiera de los dos pagos, tendrá
+                  que presentar una declaración de impuestos de 2020 para poder recibirlo.
+              - title: "¿Cómo puedo determinar si el IRS ha enviado mi cheque de estímulo?"
                 content_html: >-
-                  Si no estas seguro(a) si el IRS envió tus pagos o si quieres revisar el estatus, puedes
-                  buscarlo en la pagina de web del
+                  Si no está seguro de si el IRS ha enviado su tercer pago de estímulo o quiere comprobar el
+                  estado, puede buscar en el sitio web del IRS
                   <a
                     href="https://www.irs.gov/es/coronavirus/get-my-payment"
                     rel="nofollow"
                     target="_blank"
                     data-track-click="stimulus-irs-get-my-payment"
                     data-track-attribute-position="stimulus-how-faq2">
-                  IRS llamada Obtener Mi Pago de Impacto Económico</a>.
-              - title: "¿Qué pasa si hay un problema con mi información de cuenta bancaria?"
-                content_html: >-
-                  Si antes proporcionaste información de una cuenta bancaria al IRS y esta información ha cambiado
-                  o no reconoces el número registrado en el sistema, por favor consulta
-                  <a href="https://www.irs.gov/es/newsroom/irs-statement-update-on-economic-impact-payments"
+                  Get My Payment</a>. Para
```

Comment:
Should "Get My Payment"  be translated to spanish?

Reply:
Good call, updated it!